VSDiagnostics/VSDiagnostics/VSDiagnostics.Test/Tests/General/ImplementEqualsAndGetHashCodeTests.cs

```diff
@@ -1313,8 +1373,41 @@ public override int GetHashCode()
     }
 }";
 
-            VerifyDiagnostic(original, string.Format(ImplementEqualsAndGetHashCodeAnalyzer.Rule.MessageFormat.ToString(), "MyClass"));
+            VerifyDiagnostic(original, string.Format(ImplementEqualsAndGetHashCodeAnalyzer.Rule.MessageFormat.ToString(), "Class", "MyClass"));
             VerifyFix(original, result);
         }
+
+        [TestMethod]
+        public void ImplementEqualsAndGetHashCode_ExplicitlyCheckMessage_Class()
```

Comment:
This shouldn't be a separate test -- every preceding test just has to do it explicitly.